spec/enterprise/services/sla/evaluate_applied_sla_service_spec.rb

```diff
@@ -28,9 +28,13 @@
       it 'updates the SLA status to missed and logs a warning' do
         allow(Rails.logger).to receive(:warn)
```

Comment:
lets move the notification creation checks from this spec file into sla event model spec. 
Would simplify the specs much more and we would only need to write just the condition whether notifications are created on the event creation